2020_RTSS_cpc_model_based_algorithm/src/main.rs

```diff
@@ -24,9 +27,21 @@ struct ArgParser {
 fn main() {
     let arg: ArgParser = ArgParser::parse();
     let mut dag = create_dag_from_yaml(&arg.dag_file_path);
-    let number_of_cores = arg.number_of_cores;
+    let homogeneous_processor = HomogeneousProcessor::new(arg.number_of_cores);
     prioritization_cpc_model::assign_priority_to_cpc_model(&mut dag);
-    let mut fixed_priority_scheduler =
-        FixedPriorityScheduler::new(&dag, &HomogeneousProcessor::new(number_of_cores));
-    let (_schedule_length, _) = fixed_priority_scheduler.schedule();
+    let mut fixed_priority_scheduler = FixedPriorityScheduler::new(&dag, &homogeneous_processor);
+
+    let (schedule_length, _) = fixed_priority_scheduler.schedule();
+    let now: DateTime<Utc> = Utc::now();
+    let date = now.format("%Y-%m-%d-%H-%M-%S").to_string();
+    let file_name = format!("{}-cpc-result", date);
+    let file_path = create_yaml_file(&arg.output_dir_path, &file_name);
```

Comment:
This process should be cut out to a shared library.

Reply:
Implemented as create_log_yaml_file in output_log.rs
Output format was year-date-time-input_name-log.